src/main/java/com/google/cloud/tools/appengine/cloudsdk/CloudSdkAppEngineFlexibleStaging.java

```diff
@@ -73,16 +75,17 @@ public void stageFlexible(StageFlexibleConfiguration config) throws AppEngineExc
     }
   }
 
-  private static String findRuntime(StageFlexibleConfiguration config) {
+  @VisibleForTesting
+  static String findRuntime(StageFlexibleConfiguration config) {
     // verification for app.yaml that contains runtime:java
     Path appYaml = config.getAppEngineDirectory().toPath().resolve(APP_YAML);
     String runtime = null;
     try {
       if (Files.isRegularFile(appYaml)) {
         runtime = new AppYaml(appYaml).getRuntime();
       }
-    } catch (IOException e) {
-      log.warning("Unable to determine runtime: error parsing app.yaml : " + e.getMessage());
+    } catch (IOException | ScannerException | ParserException ex) {
```

Comment:
Note that `findRuntime()` is an internal method. From the perspective of clients calling `stageFlexible()`, I think it makes sense for `stageFlexible()` to just stage `app.yaml` regardless of the parsing failure.

If we do want the method to propagate the parsing failure, I think we will want to make `stageFlexible()` throw `AppEngineException` as other usual public methods do, but I don't really feel that's needed. Thoughts?